pkg/operator/starter.go

```diff
@@ -42,6 +42,7 @@ func RunOperator(clientConfig *rest.Config, stopCh <-chan struct{}) error {
 
 	operatorConfigInformers := operatorclientinformers.NewSharedInformerFactory(operatorConfigClient, 10*time.Minute)
 	kubeInformersNamespaced := informers.NewFilteredSharedInformerFactory(kubeClient, 10*time.Minute, targetNamespaceName, nil)
+	coreInformersNamespaced := informers.NewFilteredSharedInformerFactory(kubeClient, 10*time.Minute, operatorNamespaceName, nil)
```

Comment:
update to the construction sanchezl has in 74.

Make names consistent with `kubeInformersForOpenshiftControllerManager`